code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -389,3 +389,126 @@
 		user.stats.changeStat(stat, -max(round(stat_gain/2),1))
 	odditys.Add(I)
 	return TRUE
+
+/datum/ritual/cruciform/priest/confirmation
+	name = "Confirmation"
+	phrase = "Misericordia et veritas non te deserant circumda eas gutturi tuo et describe in tabulis cordis tui..."
+	desc = "Ritual of assigning a disciple to specific duty within the church."
+	power = 80
+	cooldown = TRUE
+	cooldown_time = 1 MINUTE
+
+/datum/ritual/cruciform/priest/confirmation/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/C,list/targets)
+	var/obj/item/implant/core_implant/cruciform/CI = get_implant_from_victim(user, /obj/item/implant/core_implant/cruciform)
+
+	if(!CI || !CI.wearer || !ishuman(CI.wearer) || !CI.active)
+		fail("Cruciform not found",user,C)
+		return FALSE
+
```

Comment:
Add a check to be sure that the target is neither a priest nor an inquisitor since it shouldn't be used on them.